### ISSUE TITLE
Player and monster intangibility - every thing became "ghosts"

### ISSUE THREAD

**Issue by kimjetwav** [chocolate-doom/chocolate-doom#1707]
### Background

Version of Chocolate Doom: 3.0.1

Operating System and version: Debian GNU/Linux trixie/sid - Chocolate Doom installed via GNU Guix.

Game: Doom 1.9. The WAD is for DOOM, not Ultimate or Final DOOM.

Any loaded WADs and mods: `chocolate-doom -setmen dos622 -strictdemos -iwad DOOM.WAD`
(iwad sourced via DOOMWADDIR envvar from user homedir)

### Bug description

Observed behavior:

Loaded E1M1 on UV, and began playing as normal. Killed one shotgunner and found that the second shotgunner was seemingly unhittable. At first I thought I was just incredibly unlucky, but as he approached me I noticed he couldn't hit me either.

It turns out both of us were intangible ("ghosts"): the shotgunner was able to pass through me, and through a pillar in the opening room, and neither of us could be shot. I also noticed a shotgunner from across the courtyard outside the window had walked through his wall and was crossing the courtyard towards me. At this point I saved the game, and quit. Unfortunately I didn't have the wherewithal to check the terminal scroll for anything unusual, as I immediately reloaded to test another run.

On reload, the saved game seemingly exhibits "normal" physics, and the only evidence I can muster for this is that the save includes two displaced shotgunners, one in the courtyard who left the nukeage room, and another stuck inside a wall in the courtyard. New games also seem to work as usual.

Can provide the save file, WAD, conf files, binaries, or other files on request. Wasn't recording a demo at the time. As far as I recall I didn't do anything out-of-the-ordinary while starting or the game, though I don't well remember as I was, you know, booting up DOOM for the 20 millionth time and wasn't thinking about it.

I was perusing DoomWiki looking for bugs and this sounds somewhat similar to the all-ghosts bug? But I was just shooting up the stairs with a pistol towards the armour platform as usual, which doesn't sound like it could trigger an intercept overflow condition as the wiki suggests would cause all-ghosts.

Expected behavior:

Expected behaviour would be that the shotgunners and player would not suddenly become intangible in this particular situation.

**Comment by fragglet** [comment]
Might be [this](https://doomwiki.org/wiki/All-ghosts_effect) but I've never heard of it happening on Doom E1M1.

**Comment by kimjetwav** [comment]
Yeah, that's what I was looking at. I know Chocolate Doom does try to preserve behaviour like that, and I couldn't see any other "classic" bug that made sense of the situation. It hasn't recurred, and all I have of it now is a dumb savegame with a shotgunner stuck in a wall and another lost in the courtyard.

**Comment by kimjetwav** [comment]
I'm gonna close this down. I've never been able to recreate this and all the evidence I've got left of it is a single save file with two misplaced shotgunners. I'm fine with saying this is the all ghosts bug, just seen in a map that it's very rarely seen in.
